clustered/server/src/main/java/org/ehcache/clustered/server/offheap/InternalChain.java

```diff
@@ -26,8 +26,15 @@
 
   boolean append(ByteBuffer element);
 
-  boolean replace(Chain expected, Chain replacement);
+  ReplaceResponse replace(Chain expected, Chain replacement);
 
   @Override
   void close();
+
+  enum ReplaceResponse {
+    MATCH_AND_REPLACED,
```

Comment:
Could you javadoc the meanings of these response types. At least the difference between `MATCH` and `EXACT_MATCH` as the rest is fairly obvious.

Reply:
Added javadoc